administrator/language/de-DE/plg_multifactorauth_email.sys.ini

```
@@ -0,0 +1,8 @@
+; Joomla! German Translation
+; (C) 2022 Open Source Matters, Inc. <https://www.joomla.org>
+; (C) Translation 2008 - 2022 J!German <https://www.jgerman.de>
+; License GNU General Public License version 2 or later; see LICENSE.txt
+; Note : All ini files need to be saved as UTF-8
+
+PLG_MULTIFACTORAUTH_EMAIL="Multi-Faktor-Authentifizierung - Authentifizierungscode per E-Mail"
+PLG_MULTIFACTORAUTH_EMAIL_XML_DESCRIPTION="Zeitlich begrenzte, sechsstellige Authentifizierungscodes, verwenden und per E-Mail erhalten."
```

```suggestion
PLG_MULTIFACTORAUTH_EMAIL_XML_DESCRIPTION="Verwendung des zeitlich begrenzten, sechsstelligen Sicherheitscodes, der per E-Mail verschickt wird"
```

Der Punkt am Ende der Zeile entfällt, da kein vollständiger Satz